containerd/main.go

```diff
@@ -48,7 +48,12 @@ var daemonFlags = []cli.Flag{
 	cli.StringFlag{
 		Name:  "runtime,r",
 		Value: "runc",
-		Usage: "name of the OCI compliant runtime to use when executing containers",
+		Usage: "binary path of the OCI compliant runtime to use when executing containers",
```

Comment:
The description of this flag is `the name of the OCI compliant runtime to use when executing containers`.But I think it is the binary path of `the OCI compliant runtime to use when executing containers` according to `exec.Command` usage and we can not guarantee the binary can be found at PATH definitely.